docs/content/configuration/configuring_topology.md

```diff
@@ -2,6 +2,8 @@
 
 Use this information for specific configuring information when using CSI Topology with the IBM® block storage CSI driver.
 
+Using the CSI Topology feature, volume access can be limited to a subset of nodes, based on regions and availability zones. Nodes can be located in various regions within an availability zone, or across the different availability zones. Using the CSI Topology feature can ease volume provisioning for workloads within a multi-zone architecture.
```

Comment:
There are no volumes in the feature of host definition.
The Topology awareness feature integration with host definition is for defining hosts on proper storage array according to the topology zones configuration, for example nodeX is in the zone of storage array1 means that only nodeX will be defined on storage array1 since other nodes are located on other zone/s that this array1 is not in it.

Reply:
updated.
the original paragraph that i put in now is from our original def. of csi topology and what it's used for. i added in now another paragraph on how it's used together with host defintiion.